docs/userstories/ClusterNetworkPolicy/Cluster_Compliance.md

```diff
@@ -0,0 +1,12 @@
+### Summary
+
+As a cluster admin, I want all workloads to start with a network/security
+model that meets the needs of my company.
+
+### Description
+
+A platform admin may want to factor out policies that each namespace would have
+to write individually in order to make deployment and auditability easier.
+Common examples include allowing all workloads to be able to talk to the cluster
+DNS service and, similarly, allowing all workloads to talk to the logging/monitoring
+pods running on the cluster.
```

Comment:
In general I think this story needs to go into detail and provide examples regarding the actual expected behavior here, i.e _who_ (if anyone) can override these baseline polices and _how_ can they do it.

Reply:
agreed .. we need to explicitly call out whether the expectation here is that these policies are baseline? or something more strict. since its compliance, it must be strict (hence not a justification of DNPs)